quiz/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+services:
+  web:
+    container_name: quiz_web
+    image: node:19.7.0-slim
+    restart: on-failure
+    working_dir: /app
+    volumes:
+      - ./web:/app
+    ports:
+      - "3000:3000"
+    networks:
+      - proxy-web
+    command: /bin/bash -c "npm ci && npm run dev"
```

Comment:
można dodać to: stdin_open: true, aby działał hot reloading podczas developmentu, nie działa w ogóle na WSL2, więc nie należy się tym przejmować

Reply:
Na linuxie działał już w takim wydaniu mi, ale dodane